components/icon/src/AccessibilityIcon.tsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import { Icon, IconProps } from './Icon';
+
+export const AccessibilityIcon = ({
+	assistiveText = 'Accessibility',
+	copyrightYear = '2020',
+	size = 'medium',
+	overrides,
+	...props
+}: Omit<IconProps, 'icon'>) => (
+	<Icon
+		icon="AccessibilityIcon"
+		assistiveText={assistiveText}
+		copyrightYear={copyrightYear}
+		size={size}
+		overrides={overrides}
+		{...props}
+	>
+		<path
+			fill="currentColor"
+			fillRule="evenodd"
+			d="M12,0 C13.375,0 14.5,1.125 14.5,2.5 C14.5,3.875 13.375,5 12,5 C10.625,5 9.5,3.875 9.5,2.5 C9.5,1.125 10.625,0 12,0 Z M22,8 L16,8 L16,24 L13,24 L13,16 L11,16 L11,24 L8,24 L8,8 L2,8 L2,6 L22,6 L22,8 Z"
+		/>
+	</Icon>
+);
+
+AccessibilityIcon.propTypes = {
+	// ----------------------------- Warning --------------------------------
+	// | These PropTypes are generated from the TypeScript type definitions |
+	// |     To update them edit TypeScript types and run "yarn prop-types"  |
+	// ----------------------------------------------------------------------
+	/**
+	 * String to use as the `aria-label` for the icon. Set to an empty string if you
+	 * are rendering the icon with visible text to prevent accessibility label
+	 * duplication.
+	 *
+	 * Defaults to the icon name e.g. `BusinessPersonIcon` --> "Business Person"
+	 */
+	assistiveText: PropTypes.string,
```

Comment:
@Duwon98 the proptype color is missing.

Reply:
Hey Kenji,
Changes have been made. Could you have a look? @kenjishiromajp @soobin93